lesson08/assignment/vue-cli/src/components/Carousel.vue

```diff
@@ -0,0 +1,57 @@
+<template>
+<div>
+    <b-carousel
+        id="carousel-1"
+        v-model="slide"
+        :interval="10000"
+        controls
+        indicators
+        background="#ababab"
+        style="text-shadow: 1px 1px 2px #333;"
```

Comment:
Inline styles are not modern best practice. This lowers the maintainability of your code, and makes it look dated. I'd recommend finding a solution that allows styling with CSS, though I know sometimes this gets tricky with libraries.